lib/resources/dstu2/diagnostic_report.yaml

```diff
@@ -0,0 +1,17 @@
+---
+name: Diagnostic Report
+field_name_base_url: http://hl7.org/fhir/DSTU2/diagnosticreport-definitions.html#DiagnosticReport
+fields:
+
+- name: category
```

Comment:
Yes, I overlooked that, added:
https://github.com/cerner/fhir.cerner.com/pull/79/commits/fd06f9665d0579a9d318d29861a92ae869b52125